recipes/kernda/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "kernda" %}
+{% set version = "0.1.0" %}
+{% set hashtype = "sha256" %}
+{% set hash = "a0452c50d7554317ac91b8bcd87a547dd5a66ffc34ccad4e2e6e2429d0ec4fbd" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  {{ hashtype }}: {{ hash }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+  entry_points:
+    - kernda=kernda.cli:cli
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+
+test:
+  imports:
+    - kernda
+
```

Comment:
Can you add

```
command:
  - kernda -h
```

To make sure the cli is getting registered properly.  Might be "commands", I don't remember off the top of my head. Otherwise LGTM